Source/ACE.Server/Command/Handlers/DeveloperContentCommands.cs

```diff
@@ -904,6 +939,25 @@ private static void ImportSQLQuest(Session session, string sql_folder, string sq
             sql2json_quest(session, quest, sql_folder, sql_file);
         }
 
+        private static void ImportSQLSpell(Session session, string sql_folder, string sql_file)
+        {
+            if (!uint.TryParse(Regex.Match(sql_file, @"\d+").Value, out var spellId))
+            {
+                CommandHandlerHelper.WriteOutputInfo(session, $"Couldn't find spell id from {sql_file}");
+                return;
+            }
+
+            // import sql to db
+            ImportSQL(sql_folder + sql_file);
+            CommandHandlerHelper.WriteOutputInfo(session, $"Imported {sql_file}");
+
+            // clear this spell out of the cache (and everything else)
+            DatabaseManager.World.ClearSpellCache();
```

Comment:
if this is a projectile spell, and someone is changing the projectile wcid, the various spell projectile caches would also need to be cleared here

from /clearcache spell:

+WorldObject.ClearSpellCache();